src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -97,7 +98,9 @@ BOOST_AUTO_TEST_CASE(chainstatemanager)
     exp_tip = c1.m_chain.Tip();
     BOOST_CHECK_EQUAL(validated_tip, exp_tip);
 
-    // Avoid triggering the address sanitizer.
+    // Drop any events from the scheduler to avoid accessing memory that is going to be unloaded
```

Comment:
Would say "Let scheduler events finish running to avoid..." instead of "Drop any events from the scheduler to avoid..."

Reply:
Thanks, done